packages/mobile/ios/celo/Info.plist

```diff
@@ -72,6 +72,8 @@
 	<string>Adding your contacts makes it easy to send and request payments with your friends.</string>
 	<key>NSPhotoLibraryAddUsageDescription</key>
 	<string>Connecting your photo library allows you to save your code to your photos.</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>This is required for you to choose a profile picture.</string>
```

Comment:
We need `es` and `pt` translations in 
- https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/ios/celo/es-419.lproj/InfoPlist.strings
- https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/ios/celo/pt-BR.lproj/InfoPlist.strings